test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -115,13 +114,12 @@ class CloudFoundryDeployTest extends BasePiperTest {
             deployTool: 'notAvailable',
             stageName: 'acceptance'
         ])
-
-        assertTrue(jlr.log.contains('[cloudFoundryDeploy] General parameters: deployTool=notAvailable, deployType=standard, cfApiEndpoint=https://api.cf.eu10.hana.ondemand.com, cfOrg=testOrg, cfSpace=testSpace, cfCredentialsId=myCreds, deployUser=testUser'))
+        // asserts
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.